### ISSUE TITLE
How to configure Actors

### ISSUE THREAD

**Issue by Basyras** [dapr/dotnet-sdk#499]
From docs from [here](https://github.com/dapr/dotnet-sdk/blob/master/docs/get-started-dapr-actor.md#optional---override-default-actor-settings) the `ConfigureActorSettings` method is used to configure Actors but i can't find it anywhere. I tried install every Dapr nuget package but still it says such a method does not exits. Iam using v0.12.0-preview1 version. Am I doing something wrong? Was this method removed? Is there another way how to configure actors?

**Comment by rynowak** [comment]
Hi @Basyras - thanks for finding this. 

This indeed changed, and we need to update the docs. Here's a PR that updates the docs: https://github.com/dapr/dotnet-sdk/pull/501

I think the part you're looking for is here: https://github.com/dapr/dotnet-sdk/blob/0bf2f6dc2feda4f6ba0f6fcd0fe47b597204f1bc/docs/get-started-dapr-actor.md#optional---override-default-actor-settings

**Comment by Basyras** [comment]
My options does not have properties as showed in PR. Does this PR reflect shipped v0.12.0-preview1?

**Comment by rynowak** [comment]
Sorry for the confusion on this - this was accidentally made more confusing in 0.12 and has already been improved in 1.0.0-rc2. We're working on finalizing the rc2 release so if you can jump to that it might be easiler. 

Here's an example workaround in `Startup.cs`:

```
using System;
using Dapr.Actors.Runtime;
using Microsoft.AspNetCore.Builder;
using Microsoft.AspNetCore.Hosting;
using Microsoft.AspNetCore.Http;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Hosting;

namespace explain
{
    public class Startup
    {
        public void ConfigureServices(IServiceCollection services)
        {
        }

        // Add ActorRuntime as a parameter
        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ActorRuntime runtime)
        {
            // configure runtime settings before request processing starts
            runtime.ConfigureActorSettings(options =>
            {
                options.ActorIdleTimeout = TimeSpan.FromHours(10);
            });

            if (env.IsDevelopment())
            {
                app.UseDeveloperExceptionPage();
            }

            app.UseRouting();

            app.UseEndpoints(endpoints =>
            {
                endpoints.MapGet("/", async context =>
                {
                    await context.Response.WriteAsync("Hello World!");
                });
            });
        }
    }
}
```

Let me know if this isn't working and I'll try to help.
